.github/workflows/build.yml

```diff
@@ -20,12 +20,14 @@ jobs:
         with:
           fetch-depth: 0
       - name: Install pip dependencies
-        run: pip install -r requirements.txt
+        run: |
+          pip install -r requirements.txt
+          pip install -e .
       - name: Build and Test
         run: |
           pytest
           mypy pyteal
-          python3 -c "import pyteal" scripts/generate_init.py --check
+          python scripts/generate_init.py --check
```

Comment:
@jasonpaulos I opened https://github.com/algorand/pyteal/pull/225 for your consideration because Github won't allow a multi-line _suggestion_ due to the removed line.

Like the PR says, it's optional.  Only approve/merge it if you agree with the idea.  If you have hesitation, let's close it and accept this PR as is.

Reply:
@jasonpaulos Thanks for being willing to take in #225 - resolving.